src/main/java/net/sourceforge/argparse4j/inf/ArgumentParserException.java

```diff
@@ -23,6 +23,9 @@
  */
 package net.sourceforge.argparse4j.inf;
 
+import static net.sourceforge.argparse4j.internal.MessageLocalization.localizeIfPossible;
```

Comment:
Thank you.
This is just my personal preference, and it is not the end of the world with dependency cycles.
It looks like by moving a couple of classes, I can achieve this in argparse4j.
If it requires substantial changes, I'll give up.